Makefile

```diff
@@ -29,8 +29,16 @@ binary:
 binary-cross:
 	./script/binary-cli $(VERSION)
 
+
+%.yaml: %.jsonnet
+	$(KUBECFG) show -o yaml $< > $@.tmp
+	mv $@.tmp $@
+
+all-yaml: kubeless.yaml kubeless-rbac.yaml
+
 kubeless.yaml: kubeless.jsonnet
-	$(KUBECFG) show -o yaml $< > $@
+
+kubeless-rbac.yaml: kubeless-rbac.jsonnet kubeless.jsonnet
```

Comment:
Because it actually depends on kubeless.jsonnet:

```
$ egrep -n import kubeless-rbac.jsonnet
3:local k = import "ksonnet.beta.1/k.libsonnet";
6:local kubeless = import "kubeless.jsonnet";
```